src/test/java/accounts/OpenAccountPageTest.java

```diff
@@ -2,18 +2,31 @@
 
 import base.BaseTest;
 import data.UserData;
+import org.testng.annotations.BeforeTest;
 import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+import pages.AccountPage;
 import pages.HomePage;
 import pages.LogInPage;
 
 public class OpenAccountPageTest extends BaseTest {
     public LogInPage logInPage;
     public HomePage homePage;
+    public AccountPage accountPage;
 
-    @Test
-    public void openAccountPageTest() throws InterruptedException {
+    @BeforeTest
+    public void login() {
         logInPage = base.openPlayGroundPage();
         homePage = logInPage.logInPlayGround(UserData.ADMIN_USER_EMAIL, UserData.ADMIN_USER_PASSWORD);
-        Thread.sleep(5000);
+
+    }
+
+    @Test
+    public void openAccountPageTest() {
+        SoftAssert softAssert = new SoftAssert();
```

Comment:
If you move it to parent class (BaseTest) to the BeforeMethod - there won't need to use it each time in your tests.

Reply:
@Evangard done